tests/Behat/Context/Reorder/ReorderContext.php

```diff
@@ -124,23 +142,23 @@ public function iShouldBeNotifiedThatPromotionIsNoLongerEnabled(string $promotio
      */
     public function iShouldSeeExactlyNotifications(int $count): void
     {
-         assert(count($this->session->getPage()->findAll('css', '.sylius-flash-message')) === $count);
+        Assert::eq(count($this->session->getPage()->findAll('css', '.sylius-flash-message')), $count);
     }
 
     /**
      * @Then I should not see any notifications
      */
     public function iShouldNotSeeAnyNotifications(): void
     {
-        assert(count($this->session->getPage()->findAll('css', '.sylius-flash-message')) === 0);
+        Assert::eq(count($this->session->getPage()->findAll('css', '.sylius-flash-message')), 0);
     }
 
     /**
      * @Then I should not proceed to my cart summary page
      */
     public function iShouldNotProceedToMyCartSummaryPage(): void
     {
-        assert(strpos($this->session->getCurrentUrl(), '/cart') === false);
+        Assert::false(strpos($this->session->getCurrentUrl(), '/cart'));
```

Comment:
You should check if the cart summary page is open on its page by using the method `isOpen`